rust/crypto/tests/aes_ctr.rs

```diff
@@ -49,7 +49,7 @@ fn aes_ctr_long_test() -> Result<(), signal_crypto::Error> {
         let mut processed = 0;
         while processed != buf.len() {
             let remaining = buf.len() - processed;
-            let this_time = std::cmp::max(1, rng.gen::<usize>() % remaining);
+            let this_time = if remaining > 1 { rng.gen_range(1, remaining) } else { remaining };
```

Comment:
👍 I didn't realize the range-based `gen_range` was from a newer `rand` than what we're using.